portfolio/src/main/webapp/about-me.html

```diff
@@ -1,95 +1,95 @@
-                            figures like Elon Musk, and his passion to accelerate mankind's journey
-                            into the future!
-                        </li>
-                        <li>
-                            Within CS, I am most excited about artificial intelligence because
-                            of it's potential to reshape our world for better or worse.
-                        </li>
-                        <li>
-                            I am scared of being clueless in a world where technology defines
-                            everything.
-                        </li>
-                    </ol>
-                </div>
-            </div>
+      <div class="content">
+        <div class="two-columns">
+          <div class="about-me-img">
+            <img src="images/about-me-img1.jpeg" alt="profile-pic.jpeg" class="circular-portrait">
+          </div>
+          <div class="about-me-content">
+            <h2 >This is me!</h2>
+            <h4>
+              Hello! My name is Chance Lau, and I study computer science at the
+              University of California - Davis.
+            </h4>
+            <h4>
+              A list of 10 things about me:
+            </h4>
+            <ol>
+              <li>
+                I am an active member of the Taekwondo club at UC Davis--
+                I love the powerful yet elegant kicks unique to this art!
+              </li>
+              <li>
+                I love to watch K-dramas; my favorite dramas are "Goblin--
+                the Lonely and Great God" and "Hotel Del Luna."
+              </li>
+              <li>
+                I enjoy learning about new and different skills/subjects,
+                ranging from crocheting to nutrition and bodybuilding to martial
+                arts!
+              </li>
+              <li>
+                I would love to visit Tokyo and South Korea in the near
+                future:)
+              </li>
+              <li>
+                In my free time, I enjoy playing strategy games such as
+                League of Legends, chess, and Civilization.
+              </li>
+              <li>
+                My ethnicity is Hmong! Historically, my people don't have a country
+                of our own. The Hmong people lived as farmers in the remote hillsides of Laos,
+                Thailand, and Vietnam.
+              </li>
+              <li>
+                Most Hmong-Americans live in either California or Minnesota.
+              </li>
+              <li>
+                I became interested in computer science because of tech-savvy
+                figures like Elon Musk, and his passion to accelerate mankind's journey
+                into the future!I became interested in computer science because of tech-savvy
+                figures like Elon Musk, and his passion to accelerate mankind's journey
+                into the future!
```

Comment:
Looks like some text was duplicated here?

Reply:
Oops, fixed!

Reply:
ummm, what was the acronym again? :D

Reply:
Ready for another look